src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.spec.ts

```diff
@@ -419,78 +419,6 @@ describe('StripeUPEPaymentStrategy', () => {
                     expect(response).toBe(store.getState());
                 });
 
-                it('fires additional action outside of bigcommerce', async () => {
```

Comment:
Hi @BC-PabloRivera, why are you removing this test case?

Reply:
Because credit cards don't use redirects. Redirects are for a specific list of methods, for example, SOFORT, which already has this test